student/src/main/java/nl/quintor/studybits/student/entities/ClaimEntity.java

```diff
@@ -43,13 +42,17 @@
     @Column(nullable = false)
     private String signatureCorrectnessProof;
 
-    @Column(nullable = false)
-    private String issuerDid;
-
-    private String myDid;
-
-    @Column(nullable = false)
-    private String theirDid;
-
-
+    @Override
+    public String toString() {
```

Comment:
Why are you creating your own JSON string here? Can't you use Dozer for custom mapping? 
Also, don't send the Student object, but only the student id.

Reply:
This is not JSON, just a readable toString for debug logging